CONTRIBUTING.md

```diff
@@ -1,76 +1,66 @@
+When you submit a pull request, a CLA-bot automatically determines if you need to provide a CLA and decorates the pull request appropriately (e.g., label, comment). Follow the instructions provided by the bot. You only need to do this once across all repositories using our CLA.
 
-It is vital that users trust the contributors to have their best interests in mind while making decisions affecting the overall direction. This means having defined guidelines, a clear roadmap, and a predictable release schedule.
+## Guiding principle
 
-Adoption of new capabilities and features must be carefully balanced by the expressed needs of the users and the community. Change for the sake of change must be avoided.
+Owners, the steering committee, collaborators, code owners, and contributors work in concert with one another on behalf of the FAST-DNA community and prioritize the communities interests over their own.
 
-In order to best serve the community as a whole, the development, release, and work management processes must reflect these guiding principles.
+The development, release, and work management processes must reflect this principle. Accepting contributions to the project requires a review by collaborators.
 
 ## Governance
-### Steering Committee
-Membership for the Steering Committee consists of key Collaborators who have demonstrated both design and technical expertise critical to the ongoing support, maintenance, evolution, and a long term commitment to driving the project and community forward.
-
-Collaborators can be nominated to become members of the Steering Committee following the same nomination and approval model. However, Collaborators cannot nominate themselves for Steering Committee membership. A Contributor cannot be nominated and accepted into the Steering Committee without first having been nominated and accepted as a Collaborator.
 
 ### Owners
-Membership for Owners, having 'admin' access, have ultimate authority and are responsible for management, maintainenance, and operations of repositories.
-
-### Collaborators
-Membership for Collaborators, having 'write' access, have major impact on the project and participate in triaging issues, reviewing code, mentoring, and working to improve and maintain overall architectural quality.
 
-Contributors can be nominated as Collaborators by Steering Committee members. Once the nomination is approved by the Steering Committee, the invitation to become a Collaborator is extended to the individual. Assuming the individual accepts the invitation and agrees to the terms of DCO (Developer Certificate of Origin) they are granted commit-access to the project.
+*Owners* have admin access and are responsible for the management, maintenance, and operations of repositories.
```

Comment:
"Owners" and "Code Owners" are two different things.  "Owners" are administrative rights on the repository security groups.  "Code owners" are collaborators that have knowledge and expertise on a single package.  I would suggest keeping the Code Owners paragraph and but clarifying the differences.  Also, these are not repositories.  This code base is a single repository.  Though this repository has multiple packages for which various people are "Code Owners" of.